packages/components/src/DataList/components/DataListSearch/DataListSearch.css

```diff
@@ -32,6 +32,7 @@
     position: static;
     visibility: visible;
     width: auto;
+    max-width: calc(var(--base-unit) * 13);
```

Comment:
Isn't the row that holds the search using `<Grid>`? I would expect this search input to take up the prescribed column value rather than be responsible for its' own width

Reply:
We're not using `Grid` here. But we are using `display: grid;` [here](https://github.com/GetJobber/atlantis/blob/080c07cb01bf6842de106c51f167120ec2a168a9/packages/components/src/DataList/DataList.css#L32), where I could set `grid-template-columns: auto fit-content(208px);` and it would work too.

Reply:
Oh OK - I'm not super opinionated about how we do it, but it does seem like we might want to put the layout responsibilities outside of this component.